torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4151,6 +4156,18 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                # see discussion https://github.com/pytorch/pytorch/pull/47761#issuecomment-747316775
                SkipInfo('TestGradients', 'test_fn_gradgrad', device_type='cuda'),)
            ),
+    OpInfo('linalg.eigvalsh',
+           aten_name='linalg_eigvalsh',
+           dtypes=floating_and_complex_types(),
+           check_batched_gradgrad=False,
+           sample_inputs_func=sample_inputs_linalg_eigh,
+           gradcheck_wrapper=gradcheck_wrapper_hermitian_input,
+           decorators=[skipCUDAIfNoMagma, skipCUDAIfRocm, skipCPUIfNoLapack],
+           skips=(
+               # cuda gradchecks are slow
```

Comment:
Not for this PR, but we should revisit if these are still "slow" after @soulitzer's work on gradcheck performance

What do you think, @soulitzer?

Reply:
I removed the skip here, see https://github.com/pytorch/pytorch/pull/57189#issuecomment-831773746.
For other OpInfos I'll check later and remove skips in a separate PR if they are not "slow" anymore.